src/templates/dashboard.html

```diff
@@ -1,6 +1,9 @@
 {% extends "layouts/base.html" %}
 {% block title %}Dashboard{% end %}
 {% block content %}
+<script>
+var data = {% raw survey_json %};
+</script>
```

Comment:
These edits were good for prototyping and seeing the data, but these changes need to be removed from the final pr.